src/main/kotlin/no/nav/syfo/client/pdfgen/PdfGenClient.kt

```diff
@@ -85,7 +85,7 @@ class PdfGenClient(
 
     companion object {
         private const val API_BASE_PATH = "/api/v1/genpdf/isbehandlerdialog"
-        const val FORESPORSEL_OM_PASIENT_PATH = "$API_BASE_PATH/foresporselompasient"
+        const val FORESPORSEL_OM_PASIENT_PATH = "$API_BASE_PATH/foresporselompasient" // TODO: Spesifiser her også?
```

Comment:
Hmm ja, burde kanskje det? Det blir jo forskjellige formater på pdfene.

Reply:
Antagelig 🤔 Får se nærmere på det når vi skal legge inn pdfen for legeerklæring. La inn et punkt på det [her](https://trello.com/c/67oIkygZ/1450-neste-dialogmeldingstype-foresp%C3%B8rsel-om-legeerkl%C3%A6ring-ved-arbeidsuf%C3%B8rhet) .